benchmarksapps/PlatformBenchmarks/PlatformBenchmarks.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <OutputType>Exe</OutputType>
+    <LangVersion>latest</LangVersion>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <RuntimeFrameworkVersion>$(MicrosoftNETCoreApp21PackageVersion)</RuntimeFrameworkVersion>
```

Comment:
This should not be necessary, it's just to emphasize where it will only work with 2.1

Reply:
Why isn't this compatible with 2.0?

Reply:
Actually I have always thought it would only work on 2.1 ... if this is wrong then we should definitely remove this line.